src/seismometer/table/analytics_table.py

```diff
@@ -539,36 +567,34 @@ def __init__(
             value=metrics_to_display or GENERATED_COLUMNS,
             title="Performance Metrics to Display",
         )
-        self._metric_values_slider = FloatRangeSlider(
-            min=0.01,
-            max=1.00,
-            step=0.01,
-            value=metric_values or [0.2, 0.8],
-            description="Metric Values",
-            style=WIDE_LABEL_STYLE,
+        metric_values = metric_values or (0.8, 0.2)
+        self._metric_values = MonotonicProbabilitySliderListWidget(
+            names=("Metric Value 1", "Metric Value 2"), value=tuple(metric_values), ascending=False
         )
         self._group_by = Dropdown(
             options=["Score", "Target"],
             value="Score",
             description="Group By",
             style=WIDE_LABEL_STYLE,
         )
+        self._cohort_dict = MultiSelectionListWidget(cohort_dict, title="Cohorts")
```

Comment:
```suggestion
        self._cohort_dict = MultiSelectionListWidget(cohort_dict, title="Cohort Filter")
```

Since we do not display multiple cohorts, but filter down to a single intersectional cohort